Basic-Scripts/DaimondPattern.py

```diff
@@ -0,0 +1,28 @@
+
+n=int(input("enter the number of rows")) #Take input

```

Comment:
Use meaningful names instead of alphabets
Follow PEP8 Guidelines
Document your code